### DISCUSSION TITLE
move入门学习-思维导图

### DISCUSSION THREAD

**Discussion by token01** [NonceGeek/Move-Camp#205] [Materials]
导图文件

![image](https://user-images.githubusercontent.com/3217439/200757510-a1ea88da-eaa8-447c-b1ae-6d10b00222cf.png)

导图在线文档
https://www.mubucm.com/doc/3N-4P7FhrDK

如果需要源文件 可以在movefuns Dao 微信群联系到我 @abel sun

**Comment by leeduckgo** [comment]
很赞很赞👍
